config/app_config.yml.SAMPLE

```diff
@@ -162,6 +162,8 @@ default: &defaults
   # Instead of defining all details here, you can also point to an entry in database.yml.
   #shared_lists: shared_lists
 
+  # default to allow sync only on less than 200 articles
+  #shared_supplier_article_limit: 2000,
```

Comment:
please keep the default value in the config file

Reply:
ok, changed to 200